projects/client-api-react/src/index.js

```diff
@@ -445,10 +443,7 @@ function generateIframeRef({
             console.debug('no iframe', typeof (iframe), { iframe, dataset });
             return () => { };
         }
-    }, [
-        url,
-        iframeStyle, iframeClassName, iframeProps, allowFullScreen
-    ]);
+    }, []);
```

Comment:
this feels a bit risky, if there's a new iframe, we need to reconnect, right?


maybe we should do something like: if (any?) params undefined (vs ''), do nothing, to avoid premature loads. Right now it's just `if (iframe && dataset)`, which may not check enough

Maybe do a few page loads to print what's going on here?

I can imagine:
* Initial frame loads have empty params till filled out, so need to suppress
* Later loads might some reason have empty and need suppressing? unclear